include/proto.h

```diff
@@ -491,6 +492,16 @@ enum {
    SL_FLICKER_OFF      = 2,         // Turn flickering off
 };
 
+/* Special item types */
```

Comment:
The name "special" isn't doing it for me.  I don't think a reader would understand what a special object is.  This property is really only about how the object is drawn, so perhaps you can come up with a name that is more descriptive of what it does.

Reply:
Understood.  I removed all references for special in this PR.  I changed the enums to be more specific so there's no danger of misunderstanding what they're for, and how they are different than other "special" constants or properties.
`ITEM_TEXT_COLOR_NONE`, `ITEM_TEXT_COLOR_MAGIC`, `ITEM_TEXT_COLOR_RARE`, etc.